pkg/controller/node/cidr_set.go

```diff
@@ -38,10 +38,16 @@ type cidrSet struct {
 func newCIDRSet(clusterCIDR *net.IPNet, subNetMaskSize int) *cidrSet {
 	clusterMask := clusterCIDR.Mask
 	clusterMaskSize, _ := clusterMask.Size()
-	maxCIDRs := 1 << uint32(subNetMaskSize-clusterMaskSize)
+
+	var maxCIDRs int
```

Comment:
Most of the code that underlies our allocation set logic assumes small ranges.  Even though this is just node for now, we're going to have to have some guards against wide mode ranges.  Some things that allocateNex my are backed by an uncompressed bitmap in etcd.  Those will all need max cidr guards